default.nix

```diff
@@ -223,6 +223,15 @@ in with super;
       qtsvg;
   }).overrideAttrs (x: { preCheck = "export OMP_NUM_THREADS=4"; });
 
+  dkh = callPackage ./dkh {};
+
+  dftd3 = callPackage ./dft-d3 {};
+
+  libefp = callPackage ./libefp {};
+
+  pcmsolver = callPackage ./pcmsolver {};
+  pcmsolver-1_2_3 = callPackage ./pcmsolver/1_2_3.nix {};
```

Comment:
Where is pcmsolver-1.2.3 used/where is it needed?

Reply:
It is not as far as i can see. I have removed it. Just copied it from my overlay.